README.md

```diff
@@ -2,77 +2,119 @@
 
 [![CircleCI](https://circleci.com/gh/pantheon-systems/decoupled-kit-js/tree/canary.svg?style=svg)](https://circleci.com/gh/pantheon-systems/decoupled-kit-js/tree/canary)
 
-## Prerequisites
+- [Prerequisites](#prerequisites)
+- [Getting Started](#getting-started)
+- [Versions and Releases](#versions-and-releases)
+- [Reporting](#bug-reports-and-feature-requests)
+- [Further Reading](#further-reading)
 
-To setup the monorepo for development the following should be installed locally:
 
-1. Nodejs version 16 LTS. We reccomend using [nvm](https://github.com/nvm-sh/nvm)
-1. [npm](https://docs.npmjs.com/cli/v8/commands/npm) version 8.x.x
-1. [pnpm](https://pnpm.io/installation) version 7.1.0
+## Prerequisites
 
-We recommend using npm to install pnpm
+To configure the monorepo for development, the following should be installed locally:
 
-```bash
-npm install -g pnpm@7.1.0
-```
+- Nodejs version 16 LTS. We recomemnd using [nvm](https://github.com/nvm-sh/nvm)
+- npm version 8.x.x
+- The latest version of [pnpm](https://pnpm.io/installation)
 
-## Getting started
+    We recommend using npm. Run the following command to install:
 
-1. Fork this repo
-1. Clone your fork to your local machine
-1. Run `pnpm install` to install dependencies for all packages and starters
-1. Run `pnpm husky install` to install the [husky pre-commit hooks](https://github.com/pantheon-systems/decoupled-kit-js/blob/canary/.husky/pre-commit) 
+    ```bash
+    npm install -g pnpm@6.32.6
```

Comment:
This was just changed today to `pnpm@7.1.0`
The upgrade also effects the commands below so for example `pnpm --filter ./starters lint` should be `pnpm --filter './starters/**' lint`